harbor-client/src/db_models/lightning_receive.rs

```diff
@@ -110,35 +124,30 @@ impl LightningReceive {
 
     pub fn get_by_operation_id(
         conn: &mut SqliteConnection,
-        operation_id: OperationId,
+        operation_id: String,
```

Comment:
I feel like all this OperationId -> String stuff (rather than doing some enum OpId { Fed(OperationId) / Cashu(String) } is going to be annoying to maintain in the long run but it's your problem lol and unless there's like a third thing it's probably not a help so your call

Reply:
I think this is fine for now, but yeah, not perfect use of types